src/main/java/cafeboard/Board/BoardService.java

```diff
@@ -0,0 +1,42 @@
+package cafeboard.Board;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@Service
+public class BoardService {
+    private final BoardRepository REPOSITORY;
+
+    public BoardService(BoardRepository REPOSITORY) {
+        this.REPOSITORY = REPOSITORY;
+    }
+
+    public Board create( CreateBoardRequest request) {
+        return REPOSITORY.save(new Board(request.title()));
+    }
+
+    public List<ReadBoardResponse> findAll() {
+        return REPOSITORY.findAll()
+                .stream()
+                .map(board -> new ReadBoardResponse(
+                        board.getId(),
+                        board.getTitle(),
+                        board.getCreateTime()))
+                .toList();
+
+    }
+
+    public Board update(Long boardId, CreateBoardRequest request) {
+        Board board = REPOSITORY.findById(boardId).orElseThrow(() -> new NoSuchElementException("id를 찾을 수 없습니다.: " + boardId));
```

Comment:
👍 구체적인 에러 좋아요.
한 줄에 점(`.`)을 연이어 찍는 것보다 줄바꿈 한 번 해 주시면 가독성면에서 더 좋다고 여겨져요.
```suggestion
        Board board = REPOSITORY.findById(boardId)
                .orElseThrow(() -> new NoSuchElementException("id를 찾을 수 없습니다.: " + boardId));
```